combat/CombatEvents.h

```diff
@@ -4,6 +4,7 @@
 
 #include <map>
 #include <memory>
+#include <string>
```

Comment:
[Not enough](https://ptpb.pw/orl0). Maybe move to `combat/CombatEvent.h` (singular), included by this file on line 16. Both use `std::string` but  `combat/CombatEvents.h` (plural) already bootlegs `<vector>` via `combat/CombatEvent.h`.

Reply:
Also sort includes.

Reply:
now?